locksmith/src/controllers/hookController.ts

```diff
@@ -34,12 +38,8 @@ export async function subscribe(
       logger.error(intentError)
       throw intentError
     }
-
-    const expiration = new Date()
-    expiration.setSeconds(
-      expiration.getSeconds() + (hub.lease_seconds ?? 60 * 60 * 24 * 30)
-    )
-
+    // 10 days by default
+    const expiration = getExpiration(hub.lease_seconds || 864000)
```

Comment:
Let's move the `|| 864000` inside `getExpiration` and also make sure that the actual expiration is at most 90 days.

Reply:
okay, will modify and add a check for extending expiration.